packages/components/src/components/expansion-panel/array-renderer.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+const ArrayRenderer = (array, open_ids, setOpenIds) => {
+    const onArrayItemClick = id => {
+        if (open_ids.includes(id)) {
+            setOpenIds(open_ids.filter(open_id => open_id !== id));
+        } else {
+            setOpenIds([...open_ids, id]);
+        }
+    };
+
+    return (
+        <div>
```

Comment:
Can this be a `React.Fragment`? Or perhaps you could wrap an array of items with a div that has a classname.

Reply:
Yes, we can use `React.Fragment` instead as well.